### PR TITLE
Remove unnecessary else

### PR DESCRIPTION
**Pull request by guiajlopes**
Just a small improvement in the code since the else is unnecessary on this case

### CODE REVIEW

File: lib/Doctrine/ORM/Query/SqlWalker.php [hunk 1]
```diff
@@ -751,7 +751,8 @@ public function walkSelectClause($selectClause)
             foreach ($class->associationMappings as $assoc) {
                 if ( ! ($assoc['isOwningSide'] && $assoc['type'] & ClassMetadata::TO_ONE)) {
                     continue;
-                } else if ( !$addMetaColumns && !isset($assoc['id'])) {
+                }
+                if ( !$addMetaColumns && !isset($assoc['id'])) {
```
Comment (Ocramius) [inline lib/Doctrine/ORM/Query/SqlWalker.php]: Add an empty line between logical blocks

Comment (guiajlopes) [reply lib/Doctrine/ORM/Query/SqlWalker.php]: @Ocramius thanks, done!

File: lib/Doctrine/ORM/Query/SqlWalker.php [hunk 2]
```diff
@@ -751,7 +751,9 @@ public function walkSelectClause($selectClause)
             foreach ($class->associationMappings as $assoc) {
                 if ( ! ($assoc['isOwningSide'] && $assoc['type'] & ClassMetadata::TO_ONE)) {
                     continue;
-                } else if ( !$addMetaColumns && !isset($assoc['id'])) {
+                }
+                
+                if ( !$addMetaColumns && !isset($assoc['id'])) {
```
Comment (lcobucci) [inline lib/Doctrine/ORM/Query/SqlWalker.php]: @guiajlopes according to our CS we need spaces before AND after the negation operator, so this should be:

```php
if ( ! $addMetaColumns && ! isset($assoc['id'])) {
```

**Review by lcobucci — CHANGES_REQUESTED** [review]

File: lib/Doctrine/ORM/Query/SqlWalker.php (hunk 2 above)
Comment (lcobucci) [reply lib/Doctrine/ORM/Query/SqlWalker.php]: Does it make sense to turn this into a single `IF` with an `OR` instead? Like:

```php
if ( ! ($assoc['isOwningSide'] && $assoc['type'] & ClassMetadata::TO_ONE)
    || ( ! $addMetaColumns && ! isset($assoc['id']))) {
```

I mean, both are fine... just wondering 😜

Comment (guiajlopes) [reply lib/Doctrine/ORM/Query/SqlWalker.php]: @lcobucci thanks for your review, make sense to turn in the same if statement

**Review by Ocramius — APPROVED** [review]
LGTM, thanks @guiajlopes

**Pull request merged**